parsers/mcmaster-parser-2.js

```diff
@@ -102,9 +102,16 @@ function parse(text){
     return timetable;
 }
 
-fs.readFile(filename, 'utf8', function(err, data) {
-    if (err) throw err;
-    console.log(ics.buildICS(parse(data)));
-});
+var convertToCal = function(rawTimetable, directory, filename, callback){
```

Comment:
I was following the flow of the old app.js and parser connection, didnt want to break anything. I'll try that though